src/lib/features/snackbar/ui/snackbar-queue.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+  import { snackbarQueue } from '../model/store';
+  import Snackbar from './snackbar.svelte';
+
+  $: activeSnackbar = $snackbarQueue[0] || null;
+</script>
+
+{#if activeSnackbar}
+  <Snackbar snackbar={activeSnackbar} />
```

Comment:
nitpick (non-blocking): a lot of repetition in this line, I'd suggest naming the prop `options` like you named your type